indico/modules/events/abstracts/templates/abstract/public.html

```diff
@@ -216,6 +199,45 @@
     </div>
 {% endmacro %}
 
+{% macro render_attachments(abstract, management) %}
+    {% if abstract.files %}
+        <div class="files field-separator">
+            <table class="tree" id="abstract-files">
+                {% for file in abstract.files %}
+                    <tr>
+                        <td>
+                            <span class="{{ icon_from_mimetype(file.content_type, 'icon-file-filled') }}">
+                                <a href="{{ url_for('.download_attachment', file, management=management) }}">
```

Comment:
you don't need to specify `management=management` explicitly - it's added automatically when building URLs in the abstract blueprint based on the current management value in the RH (see `_add_management_flag` in the blueprint). Just didn't have time to remove it from other macros where it's not needed anymore yet, but no need to add it in more places